Setting Up Neolink For Use With Blue Iris.md

```diff
@@ -0,0 +1,125 @@
+# Setting Up Neolink For Use With Blue Iris
+
+![CI](https://github.com/thirtythreeforty/neolink/workflows/CI/badge.svg)
+
+Stock Reolink cameras do not support use with non-proprietary surveillance software such as Blue Iris. Neolink enables you to use your unsupported cameras with Blue Iris and other viewers/recorders. In this guide, you will learn how to configure your Reolink cameras and Neolink for use with Blue Iris software.
+
+**This guide applies to the following camera models:**
+- B800/D800
+- E1
+
+
+## Step One: Configuring Your Cameras
+The first thing to do to make video recording run smoothly is tweak the settings on your Reolink Cameras so that they do not conflict with Neolink.
+
+### 1. Update your Camera's firmware.
+_The cameras have software bugs too, and Reolink is constantly working to fix them. Cameras ship with older versions which have **known bugs.** It's best to eliminate any unknown parameters when setting up your cameras._
+1. Download the latest version of firmware for your camera at [Reolink's official firmware update site](https://support.reolink.com/hc/en-us/sections/360002374874-Firmware)
+2. Unzip the firmware package.
+3. Refer to [Reolink's official firmware upgrade guide](https://support.reolink.com/hc/en-us/articles/360004084333-Upgrade-Firmware-via-Reolink-Client-Windows-) for more information on how to upgrade firmware.
+
+### 2. Assign a static IP address to your cameras
+_This is the most reliable setup since Neolink cannot autodetect when a camera's IP address changes._
+1. In the Reolink PC app, login to your camera.
+2. Click "Device Settings" (the gear) -> "Network General."
+3. Change "Connection Type" from "DHCP" to "Static."
+4. Enter a static IP address compatible with your network (i.e. `192.168.1.15`)
```

Comment:
This is a tricky subject - you should refer people to a resource about how to learn a suitable IP address/subnet, since you can lose access to the camera if you give it an IP that isn't routable in your network. (What if their network is 10.0.0.0/24?)